### PR TITLE
updates to username/group name formatting across all pages, other minor

### PR DESCRIPTION
**Pull request by dsajdak**
updated information.  Includes links to new videos, new ccrsoft/2024.04, and removed panfs info

### CODE REVIEW

File: pages/faq.md [hunk 1]
```diff
@@ -76,15 +76,15 @@ There are three common reasons why you might not be able to launch OnDemand sess
 CCR's `iquota` tool will provide you quota and usage for your home directory and any shared project or global scratch directories you may also have access to.  You'll first need to authenticate with the `ccrkinit` command.  Enter your password and 6 digit one time token (OTP) when prompted, as shown here:  
 
 ```
-[ccruser@vortex:~]$ ccrkinit
+CCRusername@login:~]$ ccrkinit
```
Comment (KevinVanSlyke) [inline pages/faq.md:79]: CCRusername@login: ~$ ccrkinit

File: pages/howto/matlab.md [hunk 2]
```diff
@@ -492,7 +492,7 @@ Submit a Parallel Test Job:
 
 additionalSubmitArgs =
 
-    '--ntasks=5 --cpus-per-task=1 --ntasks-per-core=1 --mem-per-cpu=8gb --qos=general-compute --clusters ub-hpc -p general-compute --mail-type=ALL --mail-user=testuser@buffalo.edu'
+    '--ntasks=5 --cpus-per-task=1 --ntasks-per-core=1 --mem-per-cpu=8gb --qos=general-compute --clusters ub-hpc -p general-compute --mail-type=ALL --mail-user=UBITusername@buffalo.edu'
```
Comment (KevinVanSlyke) [inline pages/howto/matlab.md:495]: user=[UBITusername]@buffalo.edu

File: pages/howto/easybuild.md [hunk 3]
```diff
@@ -17,14 +17,14 @@ For example, the module `java/11.0.16` defines:
 
 You can see the environment variables defined by the `java/11.0.16` module using:  
 ```
-ccruser@vortex ~:$ module show java/11.0.16 |grep EB  
+CCRusername@login ~:$ module show java/11.0.16 |grep EB
```
Comment (KevinVanSlyke) [inline pages/howto/easybuild.md:20]: CCRusername@login:~$ module show java/11.0.16 | grep EB  

Added space after |
Changed prompt to @login:~$, would suggest a commit wide find/replace from " ~:$" to ":~$"

File: pages/howto/matlab.md [hunk 4]
```diff
@@ -448,35 +448,35 @@ Download the [CCR Matlab Integration Scripts](https://g-ac407a.8c185.08cc.data.g
...
 
 ```
-You will need to configure your user infomation into the Profile including the location to your private SSH key:
+You will need to configure your user infomation into the Profile including the location to your private SSH key stored on your computer:
 
 ```
 >> c = parcluster;
->> c.AdditionalProperties.EmailAddress = 'testuser@buffalo.edu'
->> c.AdditionalProperties.IdentityFile = '/home/testuser/.ssh/private.key'
+>> c.AdditionalProperties.EmailAddress = 'UBITusername@buffalo.edu'
```
Comment (KevinVanSlyke) [inline pages/howto/matlab.md:472]: Maybe:
[UBITusername]@buffalo.edu

Users may be more likely to confuse UBITusername and CCRusername if both are bracketed, but in this case it is a variable we want users to substitute out.

File: pages/howto/easybuild.md [hunk 5]
```diff
@@ -102,9 +102,9 @@ These are automatically loaded for you when you load the `foss/2021b` module.  W
 **When using Easybuild, do NOT use the CCR login nodes.**  Always use a [compile node](../software/building.md) or do this from a compute node in an [OnDemand desktop](../portals/ood.md#desktop-interactive-apps) session or [interactive job](../hpc/jobs.md#interactive-job-submission).  These installations can use a decent amount of disk space so we recommend you use your project directory for all software installations.  We do not recommend using your home directory as these have small quotas.  If you haven't already done so, create an `easybuild` directory in your group's project directory and set the [Easybuild build prefix](../software/building.md#building-modules-for-your-group) to that directory.  Reminder: this is just an example; you'll need to set the path name to your group's shared project directory, which may not be in `/projects/academic`      
 
 ```
-$ ssh compile  
-$ mkdir /projects/academic/groupname/easybuild   
-$ export CCR_BUILD_PREFIX=/projects/academic/groupname/easybuild  
+CCRusername@login ~:$ ssh compile
```
Comment (KevinVanSlyke) [inline pages/howto/easybuild.md:105]: `CCRusername@login:~$`

Change ` ~:$` to `:~$` for lines 105-107.

File: pages/faq.md (hunk 1 above)
Comment (dsajdak) [reply pages/faq.md]: fixed

File: pages/howto/easybuild.md (hunk 3 above)
Comment (dsajdak) [reply pages/howto/easybuild.md]: fixed formatting.  Space after | isn't necessary

File: pages/faq.md [hunk 6]
```diff
@@ -113,7 +113,7 @@ The `getfacl` command is an easy way to see the permissions of a file or directo
 
 During installation you may see an error such as `"conda.exe: error while loading shared libraries: libz.so.1: failed to map segment from shared object: Operation not permitted"`  
 
-This relates to an issue storing temporary files created during installation. Please create a temporary directory within your project or Panasas scratch directory and specify that location in the installation command. For example:
+This relates to an issue storing temporary files created during installation. Please create a temporary directory within your project or global scratch directory and specify that location in the installation command. For example:
 ```
 mkdir /projects/academic/<group_name>/condatemp
```
Comment (KevinVanSlyke) [inline pages/faq.md:118]: mkdir /projects/academic/{YourGroupName}/condatemp

Comment (dsajdak) [reply pages/faq.md]: removed this question altogether as we no longer support Anaconda

File: pages/howto/easybuild.md (hunk 5 above)
Comment (dsajdak) [reply pages/howto/easybuild.md]: Fixed this and also changed the look of the login for the compile node throughout this page as it's slightly different than the login nodes.

File: pages/faq.md [hunk 7]
```diff
@@ -113,7 +113,7 @@ The `getfacl` command is an easy way to see the permissions of a file or directo
...
 TMPDIR=/projects/academic/<group_name>/condatemp ./Anaconda3-2020.02-Linux-x86_64.sh --prefix=/projects/academic/<group_name>/<install_dir>
```
Comment (KevinVanSlyke) [inline pages/faq.md]: TMPDIR=/projects/academic/[YourGroupName]/condatemp ./Anaconda3-2020.02-Linux-x86_64.sh --prefix=/projects/academic/[YourGroupName]/[InstallDir]

We haven't discussed arbitrary software paths yet. I think for now either [InstallDir] or <install_dir> format is fine.

File: pages/howto/easybuild.md [hunk 8]
```diff
@@ -102,9 +102,9 @@ These are automatically loaded for you when you load the `foss/2021b` module.  W
 **When using Easybuild, do NOT use the CCR login nodes.**  Always use a [compile node](../software/building.md) or do this from a compute node in an [OnDemand desktop](../portals/ood.md#desktop-interactive-apps) session or [interactive job](../hpc/jobs.md#interactive-job-submission).  These installations can use a decent amount of disk space so we recommend you use your project directory for all software installations.  We do not recommend using your home directory as these have small quotas.  If you haven't already done so, create an `easybuild` directory in your group's project directory and set the [Easybuild build prefix](../software/building.md#building-modules-for-your-group) to that directory.  Reminder: this is just an example; you'll need to set the path name to your group's shared project directory, which may not be in `/projects/academic`      
 
 ```
-$ ssh compile  
-$ mkdir /projects/academic/groupname/easybuild   
-$ export CCR_BUILD_PREFIX=/projects/academic/groupname/easybuild  
+CCRusername@login:~$ ssh compile
```
Comment (KevinVanSlyke) [inline pages/howto/easybuild.md:105]: Prompts from line 106 on are ` ~ $`, I think the format in this line are more standard `:~$`

Comment (dsajdak) [reply pages/howto/easybuild.md:105]: The prompts on the compile nodes are slightly different from the login nodes

**Review by KevinVanSlyke — APPROVED** [review]

File: pages/hpc/jobs.md
Comment (KevinVanSlyke) [inline pages/hpc/jobs.md:1]: [CCRusername@login:~]$ snodes all ub-hpc/general-compute |grep gpu

Comment (dsajdak) [reply pages/hpc/jobs.md:1]: Good catch!  I did a search for vortex but missed these.  All fixed!

**Pull request merged**